### DISCUSSION TITLE
Is Vendor App same as Seller App ?

### DISCUSSION THREAD

**Discussion by yurigeorgiev4** [amzn/selling-partner-api-models#3251] [General] [answered]
In your recent announcement you mentioned Vendor App in this section

> Manage Your App
> Vendors can now view and interact with the applications they have authorized via the Manage Your App page in Vendor Central 
> at https://vendorcentral.amazon.com/apps/manage. This page also provides the ability to authorize new developers and disable  authorizations for vendor apps for all marketplaces.

If you already have a Seller Central App would it work in Vendor Central or you'd need another app for Vendors?

Thank you!

**Comment by krithic15** [comment, marked as answer]
Hello @yurigeorgiev4,

For a developer who has a seller application and is looking to develop their vendor API integration, here is a brief overview of the steps:
To use an existing app for vendor integration:
1.	Log-in to Seller Central and go to Developer Central console and click on Edit App.
2.	Choose 'Vendors' under 'Business entities supported' and then select the necessary Vendor roles.
3.	Then click on 'Save and exit'.
4.	DONE: Now you are ready to begin calling Selling Partner APIs for vendors.

You can also create a new app on Seller Central for vendor integration.

Best,
Krithi, Solutions Architect, SP API

**Comment by yurigeorgiev4** [comment]
Thank you, krithic15! That helps, appreciate it.
